packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -166,7 +167,7 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
       calloutMaxHeight! && getContentMaxHeight && calloutMaxHeight! < getContentMaxHeight
         ? calloutMaxHeight!
         : getContentMaxHeight!;
-    const overflowYHidden = !!finalHeight;
+    const overflowYHidden = !!finalHeight || hideOverflow;
```

Comment:
if hideOverflow is explicitly set to false, should it override finalHeight?

Reply:
I could set it to do so of course.  Do you have a specific use case, or just thinking its a nice door to explicitly have?